Formula/m/msolve.rb

```diff
@@ -0,0 +1,61 @@
+class Msolve < Formula
+  desc "Library for Polynomial System Solving through Algebraic Methods"
+  homepage "https://msolve.lip6.fr"
+  url "https://github.com/algebraic-solving/msolve/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "742e84cf4d11eeadf62002623ecb7658e5d6d8c838fcf571fac06acf44252983"
+  license "GPL-2.0-or-later"
+
+  head "https://github.com/algebraic-solving/msolve.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  depends_on "flint"
+  depends_on "gmp"
+  depends_on "mpfr"
+  on_macos do
+    depends_on "libomp"
+  end
+
+  def install
+    if OS.mac?
+      libomp = Formula["libomp"]
+      ENV["OPENMP_CFLAGS"] = "-Xpreprocessor -fopenmp -I#{libomp.opt_include} #{libomp.opt_lib}/libomp.dylib"
+    end
+
+    inreplace "configure.ac", /AX_EXT/, " "
+    system "autoreconf", "-vif"
+    system "./configure",
+           "--disable-dependency-tracking",
+           "--disable-silent-rules",
+           "--enable-openmp=yes",
+           "--prefix=#{prefix}",
+           "--libdir=#{lib}",
+           *std_configure_args
```

Comment:
Most of these flags are already in `std_configure_args`:
```suggestion
    system "./configure", "--disable-silent-rules", "--enable-openmp=yes", *std_configure_args
```